applications/xe-cnf/config.sh

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2021 Exium Inc.
+
+#!/bin/bash
```

Comment:
The shebang must be on the first line. 

#!/bin/bash
# SPDX-License-Identifier: Apache-2.0
# Copyright (c) 2021 Exium Inc.

Alter it like this